ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -76,33 +77,88 @@ export interface AccountCreationState {
 }
 
 const styles = {
-  sectionLabel: {
-    height: '22px',
+  asideContainer: {
+    backgroundColor: colorWithWhiteness(colors.primary, 0.85),
+    borderRadius: 8,
+    height: '17rem',
+    width: '18rem',
+    padding: '0.5rem'
+  },
+  asideHeader: {
     color: colors.primary,
-    fontFamily: 'Montserrat',
-    fontSize: '16px',
     fontWeight: 600,
-    lineHeight: '22px',
-    paddingBottom: '1.5rem'
+    fontSize: 16,
+  },
+  asideText: {
+    fontSize: 14,
+    fontWeight: 400,
+    color: colors.primary,
+  },
+  multiInputSpacing: {
+    marginLeft: '2rem'
+  },
+  publiclyDisplayedText: {
+    fontSize: 12,
+    fontWeight: 400
+  },
+  sectionHeader: {
+    width: '26rem',
+    borderBottom: `1px solid ${colors.primary}`,
+    color: colors.primary,
+    fontWeight: 600,
+    fontSize: 18,
+    marginBottom: '1rem',
   },
-  section: {
+  sectionInput: {
     width: '12rem',
     height: '1.5rem'
+  },
+  text: {
+    fontSize: 14,
+    color: colors.primary,
+    lineHeight: '22px',
   }
 };
 
 const nameLength = 80;
 
 export const Section = (props) => {
   return <FormSection
-      style={{display: 'flex', flexDirection: 'column', paddingTop: '1rem', ...props.style}}>
-    <label style={styles.sectionLabel}>
+      style={{display: 'flex', flexDirection: 'column', ...props.style}}>
```

Comment:
Can we use <FlexColumn> tag here?

Reply:
I don't think we can make it a FlexColumn, because its a FormSection not a div. I can abstract that to a style object though